src/plugins/intel_cpu/src/transformations/transformation_pipeline.cpp

```diff
@@ -470,6 +471,34 @@ void Transformations::PreLpt(const std::vector<ov::element::Type>& defaultPrecis
         },
         ov::pass::NormalizeL2Decomposition);
 
+    CPU_SET_CALLBACK_X64(manager,
+        [this](const_node_ptr &node) -> bool {
+            if (node->is_dynamic() || node->get_element_type() != element::f32 || inferencePrecision != element::f32)
+                return false;
+            const auto group_norm = ov::as_type_ptr<const ov::op::v12::GroupNormalization>(node);
+            if (!group_norm)
+                return false;
+            const auto num_groups = static_cast<size_t>(group_norm->get_num_groups());
+            const auto shape = group_norm->get_input_partial_shape(0).to_shape();
+            size_t snippets_work_amount = shape[0] * num_groups;
+            size_t concurrency = config.streamExecutorConfig.get_threads_per_stream();
```

Comment:
Unfortunately, this value is equal to zero here since `calculate_streams` is called after these passes (not before) 😢 
https://github.com/openvinotoolkit/openvino/blob/3d45a644969454d09c3f7e97ea60cbbd8abb0740/src/plugins/intel_cpu/src/plugin.cpp#L321

Reply:
yes, you are right. concurrency is always zero at this stage. let's discuss offline

Reply:
As discussed offline, the more restrict condition(parallel_get_max_threads()) is used. Thanks Alexandra!